aerogear/mobile-security-service-operator
# feat(New Type) : add new type for bind and its commands

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8920

## What

- feat(New Type) : add new type for bind and its commands
- fix(readme) : Fix wrong and outdated information
- imp(makefile) : Rename command and fix msgs to make more clear what they are doing

# How
Following the commands used to create the Type. 

```
$ operator-sdk add api --api-version=mobile-security-service.aerogear.com/v1alpha1 --kind=MobileSecurityServiceBind
$ operator-sdk generate k8s
```
## Why
- Make easier work with and use it
- The new type is required for the binding and unbinding actions

## Verification Steps
1. Check that all still working after the changes (`make create-all and make delete-all`)
2. Test that was possible apply the new type (`make apply-bind-crd` and `make deletee-bind-crd`)

## Checklist:


## Progress
